experiments/long_runs/paper_plots.jl

```diff
@@ -0,0 +1,530 @@
+using ClimaUtilities.ClimaArtifacts
+import ClimaDiagnostics
+import ClimaAnalysis
+import ClimaAnalysis.Visualize as viz
+import ClimaUtilities
+import ClimaUtilities.ClimaArtifacts: @clima_artifact
+import ClimaParams as CP
+using ClimaLand
+using ClimaLand.Snow
+using ClimaLand.Soil
+using ClimaLand.Canopy
+import ClimaLand
+import ClimaLand.Parameters as LP
+using CairoMakie
+import GeoMakie
+using Dates
+using Statistics
+
+root_path = joinpath(pwd(), "snowy_land_longrun_gpu")
+!isdir(root_path) && mkdir(root_path)
+# outdir = "snowy_land_longrun_gpu/output_active" # on local
+# Set outdir to wherever diagnostics are saved
+# outdir = "snowy_land_longrun_gpu-3720-a_larger2_ksat_alpha/output_active" # on local
+# outdir = "snowy_land_longrun_gpu-3761-alpha1/output_active" # on local
+# outdir = "snowy_land_longrun_gpu-3771-new_default/output_active"
+# outdir = "snowy_land_longrun_gpu-3761-new_default/output_active" # on local
+outdir = "snowy_land_longrun_gpu-3785"
+root_path = outdir
+
+short_names = ["lhf", "shf", "lwn", "swn"]#, "lwu", "swu"]
+title_stubs = Dict(
+    "lhf" => "LE",#"Latent heat flux",
+    "shf" => "H",#"Sensible heat flux",
+    "lwu" => "LWᵤ",#"Upward longwave radiation",
+    "swu" => "SWᵤ",#"Upward shortwave radiation",
+    "lwn" => "LWₙ",#"Net longwave radiation",
+    "swn" => "SWₙ",#"Net shortwave radiation",
+)
+# Define levels for contour colorbars
+levels_dict = Dict(
+    "lhf_bias" => collect(-50:10:30),
+    "shf_bias" => collect(-30:10:50),
+    "lwu_bias" => collect(-40:5:20),
+    "swu_bias" => collect(-20:5:40),
+    "lwn_bias" => collect(-40:5:40),
+    "swn_bias" => collect(-40:5:40),
+)
+
+include("data_paper_plots.jl")
+
+function compute_global_average(masked_var)
```

Comment:
As of recent ClimaAnalysis (need a release after 0.5.14), we can use `weighted_average_lonlat(masked_var)` instead of this function